testfile/test.go

```diff
@@ -1,10 +1,12 @@
 package test
 
-func test() {
-		//testerr 111111
-}
-
-
 func test2(){
+		//testerr 33333
+
+//222222222
 			//testerr 33333
+
+
+
+//rrrrrrrrr
 }
```

Comment:
[gofmt]  Is your code not properly formatted? Here are some suggestions below
```suggestion 
 func test2() { 
 	//testerr 33333 
  
 	//222222222 
 	//testerr 33333 
  
 	// rrrrrrrrr